src/components/checkoutpage.jsx

```diff
@@ -0,0 +1,297 @@
+import React, {Component} from 'react';
+import { Col, Button, Form, FormGroup, Label, Input, Row, Container, Breadcrumb, BreadcrumbItem} from 'reactstrap';
+import SectionHeadingAndWhiteLine from './complibrary/sectionheadingandwhiteline';
+import UpdateCart from './complibrary/updatecart';
+
+class CheckoutPage extends Component {
+
+    constructor(...args){
```

Comment:
same as in https://github.com/reetom/smartcomm/pull/1/files#diff-2510a48b3e56f1aca24409927584f1d9